upload-prompts/config.yml

```diff
@@ -6,7 +6,10 @@ categories:
 steps:
 - title: Introduction
   content: |
-    Programmatically uploading prompts is done in three steps: 1) creating the prompt object, 2) creating the prompt's resource, and 3) uploading the prompt's media file.
+    Programmatically uploading prompts is done in three steps: 
+      1) creating the prompt object, 
```

Comment:
Capitalize each of the letters for each number.

Reply:
+Capitalized letters